exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -83,56 +84,77 @@ public String getContent() {
         maxRecords = Math.max(sp.getRecords(), maxRecords);
       }
 
-      builder.appendFormattedInteger(maxBatches, null);
-      builder.appendFormattedInteger(maxRecords, null);
-      builder.appendBytes(op.getPeakLocalMemoryAllocated(), null);
+      builder.appendFormattedInteger(maxBatches, null, null);
+      builder.appendFormattedInteger(maxRecords, null, null);
+      builder.appendBytes(op.getPeakLocalMemoryAllocated(), null, null);
     }
     return builder.build();
   }
 
-  public static final String[] OPERATORS_OVERVIEW_COLUMNS = {"Operator ID", "Type", "Min Setup Time", "Avg Setup Time",
-    "Max Setup Time", "Min Process Time", "Avg Process Time", "Max Process Time", "Min Wait Time", "Avg Wait Time",
-    "Max Wait Time", "Avg Peak Memory", "Max Peak Memory"};
+  public static final String[] OPERATORS_OVERVIEW_COLUMNS = {"Operator ID", "Type",
+    "Avg Setup Time", "Max Setup Time",
+    "Avg Process Time", "Max Process Time", "Min Wait Time", "Avg Wait Time", "Max Wait Time",
+    "% Fragment Time", "% Query Time", "Rows",
+    "Avg Peak Memory", "Max Peak Memory"};
```

Comment:
Not sure if we need to make a finalized set of string, since the array itself is finalized. Only in case of the Fragment Overview, would it make sense since the same text is used in two places.